.github/workflows/tests.yaml

```diff
@@ -43,15 +44,18 @@ jobs:
         env:
             BUILD_VARIANT: ${{matrix.build_variant}}
             CHIP_TOOL_VARIANT: ${{matrix.chip_tool}}
-            TSAN_OPTIONS: "halt_on_error=1 suppressions=scripts/tests/chiptest/tsan-linux-suppressions.txt"
+            TSAN_OPTIONS:
+                "halt_on_error=1
+                suppressions=scripts/tests/chiptest/tsan-linux-suppressions.txt"
```

Comment:
And these?  It looks like you have some auto-formatter running that has different opinions from the existing formatting.  Please get rid of the spurious changes, so this can be reviewed sanely without minutely double-checking all the thigns that got munged.

Reply:
Oh sorry! Those changes sneaked in. It's because there is a prettier config in the project (see [here](https://github.com/project-chip/connectedhomeip/blob/master/.prettierrc.json)) and it's applied to this file. Which actually seems reasonable since there is such config and im wondering why this is not applied for others too 😅

Reply:
I have no idea what that prettier config is, but clearly it's not used by the people who are normally editing these files....  ;)

Reply:
Prettier is a code formatter tool which is pretty awesome and its in the project since 3 years :D 
That being said pushed revert of those changes 🙂